source/develop/release-management/features/storage/incremental-backup.html.md

```diff
@@ -199,47 +205,47 @@ Response:
 <disks>
     <disk>
         ...
-        <backup>incremental|null</backup>
+        <backup>incremental|none</backup>
         ...
-    </disks>
+    </disk>
     ...
 </disks>
 ```
 
 #### Starting backup
 
-Start incremental backup since backup id "previous-backup-uuid".
+Start incremental backup since checkpoint id "previous-checkpoint-uuid".
 
 Request:
 ```
 POST /vms/vm-uuid/backups
 
-<backup>
-    <incremental>previous-backup-uuid</incremental>
+<vm_backup>
+    <checkpoint>previous-checkpoint-uuid</checkpoint>
```

Comment:
```<checkpoint>``` is confusing. What is wrong with ```<incremental>``` used by libvirt?

```<incremental>``` has the nice properly of telling that the backup is incremental.

Reply:
The properties in the rest-api are usually mapped to engine entities. In the engine we only have a notion of checkpoints, and 'incremental' refers to a checkpoint. So it seems more confusing to use 'incremental', It isn't really a property that implies a checkpoint id.. maybe 'incremental_checkpoint' or 'incremental_id'?